pkg/cliplugins/workspace/plugin/syncer.yaml

```diff
@@ -23,7 +23,7 @@ rules:
   resources:
   - namespaces
   verbs:
-  - create
+  - "*"
```

Comment:
Why star? What is missing?

Reply:
Updated to `create`, `list` and `watch` which are required. `delete` will also be required when the syncer gainst he ability to respond to deletion.

Reply:
please add a TODO (and a security+timebomb issue) that the syncer has to get rid of namespace permission. These mean basically admin access to the pcluster. 

We have had some conversation before that we will need some kind of privilege separation. Something to think about: would project+projectrequest be enough in an OpenShift cluster to enable syncer namespace creation?

Reply:
Issues that touch on this already:

 - https://github.com/kcp-dev/kcp/issues/387
 - https://github.com/kcp-dev/kcp/issues/276

I think there needs to be a namespace agent separate from the syncer that is deployed to each cluster and is responsible for creating/updating/deleting pcluster namespaces and managing the role for each syncer deployed to the pcluster.